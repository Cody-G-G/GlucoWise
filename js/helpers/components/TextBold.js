'use strict';
import React from 'react';
import {Text} from 'react-native';
export default (props) => <Text style={{fontWeight: 'bold'}}>{props.children}</Text>;