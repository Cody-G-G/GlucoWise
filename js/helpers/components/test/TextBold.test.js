'use strict';
import React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';
import TextBold from "../TextBold";

test('TextBold - renders correctly', () => {
    const style = {
        fontSize: 25,
        color: 'black',
        fontWeight: 'bold',
        flex: 2.5,
        textAlign: 'center'
    };

    const tree = renderer.create(
        <TextBold style={style}/>
    ).toJSON();

    expect(tree).toMatchSnapshot();
});