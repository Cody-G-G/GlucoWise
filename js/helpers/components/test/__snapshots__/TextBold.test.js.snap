// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextBold - renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "black",
      "flex": 2.5,
      "fontSize": 25,
      "fontWeight": "bold",
      "textAlign": "center",
    }
  }
/>
`;
